@Goodnews-code/gitcollab-cmd-task.md

```diff
@@ -7,7 +7,7 @@ Hi, I'm @Goodnews-code. I'm currently learning to collaborate with Git and GitHu
 Unordered list used for listing markdown examples below:
 
 - Link: [This link text will take you to Collabo Community's Github Page when you click on it!](https://github.com/collabo-community)
-- Inline Code Block: `This code block text will show up inline`.
+- Inline Code Block (with non-code text): `This code block text will show up inline.` This part of the text is just ordinary text!
```

Comment:
Here's the next step for you to fix the conflict in this area using git command line @Goodnews-code 

#

**What to do next?** 
Here is the correct content that should be on the "Inline Code Block (with non-code text)" line:
````
- Inline Code Block (with non-code text): `This code block text will show up inline.` This part of the text is just ordinary text! Do you notice the difference?
````
- Simply follow the directions given in this wiki resource (to merge your conflict locally via command line git): https://github.com/collabo-community/good-first-issue-solutions/wiki/GIT-&-GITHUB-%7C-Merging-conflicts-using-git-command-line
- Request for review as usual, once you are done. If you get lost or need help at any point, get help on the appropriate channel in our discord community.